index.src.html

```diff
@@ -513,20 +514,24 @@ <h3 id="fetch-integration">Fetch Integration</h3>
 
   ISSUE: Monkey patching! Talk with Anne.
 
-  If the <a>`Clear-Site-Data`</a> header is present in an HTTP
-  <a>response</a>, then data MUST be cleared before rendering the response to
-  the user. That is, before step #9 in the current <a>main fetch</a> algorithm,
-  execute the following step:
+  If the <a>`Clear-Site-Data`</a> header is present in an HTTP <a>response</a>
+  received from the network, then data MUST be cleared before rendering the
```

Comment:
Would you mind adding a note somewhere (perhaps in a "Service Workers" section under "Security Considerations") that outlines the rationale for only accepting responses from the network, and not those from Service Workers (e.g. cached responses, synthesized responses, etc).